packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,155 @@
+
+```html
+<nimble-accordion appearance="block">
+    <nimble-accordion-item>
+        <div slot="heading">Accordion one</div>
+        <nimble-checkbox></nimble-checkbox>
+        Accordion one content
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <div slot="heading">Accordion two</div>
+        <nimble-button>Text Button</nimble-button>
+        Accordion two content
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <div slot="heading">Accordion three</div>
+        <nimble-checkbox></nimble-checkbox>
+        <nimble-accordion>
+            <nimble-accordion-item>
+                <div slot="heading">Nested Accordion One</div>
+                Nested Accordion One Content
+            </nimble-accordion-item>
+            <nimble-accordion-item>
+                <div slot="heading">Nested Accordion Two</div>
+                Nested Accordion Two Content
+            </nimble-accordion-item>
+        </nimble-accordion>
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <div slot="heading">Accordion four</div>
+        <nimble-button>Text Button</nimble-button>
+        Accordion four content
+    </nimble-accordion-item>
+</nimble-accordion>
+```
+
+### API
+
+#### Nimble Accordion
+
+[FAST accordion API](https://github.com/microsoft/fast/blob/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion/accordion.spec.md)
+
+-   _Component Name:_ `nimble-accordion`
+-   _Properties/Attributes:_
+    -   `appearance` - `block`, `outline`, `ghost`
+    -   FAST's attribute `expand-mode` will be omitted from the FAST base class due to issues discussed below.
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ Unchanged
+-   _Slots:_ Unchanged
+
+The design includes appearances of the accordion header that reflect those of the `block`, `outline`, and `ghost` appearances used in other components. These designs are similar to those of the nimble-button and its `appearance` types, but the accordion does not use the `.control` css class, it uses `.region` and `.heading`. Because of this, differences in border functionality, and the possibility of excess overridden css classes if patterns are used, shared styles will not be used, and new styles for the accordion and accordion-item will be created. The appearances of `block`, `outline`, and `ghost` will be implemented under the HTML attribute `appearance`, which will use conditional css styles based on `appearance`'s value (ex. `:host([appearance='outline'])` will have styling for the outline button).
+
+Documentation will be added to advise against using multiple Nimble Accordions with different appearances next to each other.
+
+When using nested accordions, the parent accordion will have the selected `appearance` type, and all children accordions will be forced to have the `ghost` appearance type.
```

Comment:
I haven't been following along very closely to this HLD, so I might have missed the discussion on this -- why do all child accordions need to be forced to have a `ghost` appearance type? It would be good to have a sentence justifying/explaining why this is the case.